src/Command/RoutesCommand.php

```diff
@@ -61,6 +61,12 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
             }
 
             $output[] = $item;
+
+            // Count route templates
+            if (!array_key_exists($route->template, $duplicateRoutesCounter)) {
```

Comment:
```suggestion
            if (!isset($duplicateRoutesCounter[$this->template])) {
```
Since the value is never set to `null`, `isset()` is good enough.

Reply:
ok, thx 😄

Reply:
I adjusted it again to `$route->template` instead if your recommended `$this->template` because otherwise it would have tried to read the property `template` from the RoutesCommand object 😅